java/libphonenumber/src/com/google/i18n/phonenumbers/ShortNumberInfo.java

```diff
@@ -544,12 +543,12 @@ private boolean matchesEmergencyNumberHelper(String number, String regionCode,
       return false;
     }
     PhoneMetadata metadata = MetadataManager.getShortNumberMetadataForRegion(regionCode);
-    if (metadata == null || !metadata.hasEmergency()) {
+    if (metadata == null || metadata.emergency == null) {
```

Comment:
Actually, the default value is null. See the nano/Phonemetadata.java file line 426 below.